src/picongpu/include/traits/PICToAdios.tpp

```diff
@@ -60,6 +60,15 @@ namespace traits
         type(adios_unsigned_long) {}
     };
 
+    template<>
+    struct PICToAdios<uint64_cu>
```

Comment:
We can't cast our data befor we write because it is a generic method.
But as wrote in our chat session we can test if it is possible to define `struct PICToAdios<unsigned long int>` and `struct PICToAdios<unsigned long long int>`.